@phunkyfish 
besides that warning it compiles fine at Windows, runtime looks working well
```
compiling diskwriter_windows.cpp
downloadmanager.cpp: In member function 'void DownloadManager::handleGetFinished(QNetworkReply*)':
downloadmanager.cpp:79:43: warning: this statement may fall through [-Wimplicit-fallthrough=]
   79 |             redirectionUrl = reply->header(QNetworkRequest::LocationHeader).toUrl();
      |                              ~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
downloadmanager.cpp:80:9: note: here
   80 |         case RESPONSE_REDIRECT:
      |         ^~~~
```

--- a/downloadmanager.cpp
+++ b/downloadmanager.cpp
@@ -77,6 +77,10 @@ void DownloadManager::handleGetFinished(QNetworkReply *reply)
         switch (responseCode) {
         case RESPONSE_FOUND:
             redirectionUrl = reply->header(QNetworkRequest::LocationHeader).toUrl();
+            // Fallthrough to redirect is required, hence no break;
+#if (defined(_MSVC_LANG) && _MSVC_LANG >= 201703L) || (defined(__cplusplus) && __cplusplus >= 201703L)
+            [[fallthrough]];
+#endif
         case RESPONSE_REDIRECT:
             if (redirectionUrl.isValid()) {
                 qDebug() << reply->url() << "redirected to" << redirectionUrl;

